protocol/chat.go

```diff
@@ -218,6 +218,9 @@ type ChatPreview struct {
 
 	ContentType protobuf.ChatMessage_ContentType `json:"contentType,omitempty"`
 
+	// ImageLocalURL is the local url of the image
+	ImageLocalURL string `json:"imageLocalUrl,omitempty"`
```

Comment:
```suggestion
	// LastMessageImageLocalURL is the local url of the chat's last message's image 
	LastMessageImageLocalURL string `json:"lastMessageImageLocalURL,omitempty"`
```

Naming stuff is a pain, I don't think my suggestion is an improvement, I'm just trying to explain a little better what this field is.